Lib/fontTools/subset.py

```diff
@@ -1254,6 +1254,23 @@ def collect_features(self):
     return _uniq_sort(sum((s.Script.collect_features()
                            for s in self.ScriptRecord), []))
 
+# CBLC will inherit it
+@_add_method(ttLib.getTableClass('EBLC'))
+def subset_glyphs(self, s):
+  for strike in self.strikes:
+    for indexSubTable in strike.indexSubTables:
+      indexSubTable.names = [n for n in indexSubTable.names if n not in s.glyphs]
```

Comment:
@rsheeter isn't there a contradiction here? `s.glyphs` is meant to be the list of glyph names which one wants to *retain*. So here you would want to keep all the names which are in `s.glyphs`, and not those which are not...

```diff
-indexSubTable.names = [n for n in indexSubTable.names if n not in s.glyphs]
+indexSubTable.names = [n for n in indexSubTable.names if n in s.glyphs]
```